config/flipt.schema.cue

```diff
@@ -89,10 +89,15 @@ import "strings"
 		ttl?:     =~#duration | int | *"60s"
 
 		redis?: {
-			host?:     string | *"localhost"
-			port?:     int | *6379
-			db?:       int | *0
-			password?: string
+			host?:                 string | *"localhost"
+			port?:                 int | *6379
+			use_tls?:              bool | *false
+			db?:                   int | *0
+			password?:             string
+			pool_size?:            int | *0
+			min_idle_connections?: int | *0
```

Comment:
```suggestion
			min_idle_conn?: int | *0
```

just so that its consistent with our existing `conn` related values? https://www.flipt.io/docs/configuration/overview#database

Reply:
done